crates/common/tedge_config/src/tedge_config_cli/new.rs

```diff
@@ -382,7 +392,12 @@ define_tedge_config! {
             #[tedge_config(example = "true")]
             #[tedge_config(default(value = true))]
             timestamp: bool,
-        }
+        },
+
+        /// Set of MQTT topics the AWS IoT mapper should subscribe to
+        #[tedge_config(example = "tedge/measurements,tedge/measurements/+")]
+        #[tedge_config(default(value = "tedge/measurements,tedge/measurements/+,tedge/alarms/+/+,tedge/alarms/+/+/+,tedge/events/+,tedge/events/+/+,tedge/health/+,tedge/health/+/+"))]
```

Comment:
`tedge/health/+/+` for which component's health status?.
I think it must be `tedge/health & tedge/health/+`.

Reply:
No, either tedge/health/<service> or tedge/health/<child>/<service>.